packages/react-reconciler/src/ReactFiberWorkLoop.js

```diff
@@ -2614,7 +2655,10 @@ function commitRootImpl(
     }
 
     if (enableSchedulingProfiler) {
-      markCommitStopped();
```

Comment:
You could make these call sites a bit cleaner but not renaming the function but introducing another module that exposes all these functions and internally does:
```
if (enableSchedulingProfiler) {
  markCommitStoppedInDevTools();
}
if (enableTracingHooks) {
markCommitStoppedInTracingHooks(root);
}
```
Or have one of the two module delegate to the other one. It would reduce the cognitive complexity of this in core files like this one.

Reply:
I did it this way to avoid coupling it too much with devtools, as some parameters are specific for this API. I'm happy to refactor this if there are no concerns about that.